docker/centrifuge-chain/Dockerfile

```diff
@@ -4,13 +4,13 @@
 FROM --platform=linux/amd64 docker.io/paritytech/ci-linux:production as builder
   # Defaults
 	ENV RUST_BACKTRACE 1
-  
+
   # RustUp
 	COPY ./scripts scripts
 	COPY rust-toolchain.toml ./
 	# RUN rustup update && rustup default
 	ENV PATH=~/.cargo/bin:$PATH
-	RUN	scripts/install_toolchain.sh 
+	RUN	scripts/install_toolchain.sh
```

Comment:
Nit - remove extra spaces ^^

Reply:
Actually, I think the change is removing the space, am I wrong?

Reply:
Was referring to the space between `RUN` and `scripts/...` . It shows up weird on my diff here, nvm this if it looks OK in your IDE.

Reply:
Ah, this was added intentionally. Looks like all sections in the scripts are split by a space except that section.